grammars.yml

```
@@ -1144,6 +1144,8 @@ vendor/grammars/vscode-singularity:
 - source.singularity
 vendor/grammars/vscode-slice:
 - source.slice
+vendor/grammars/vscode-vba-json:
+- source.vba
```

Looks like you might have modified this file directly. There is no need as it is maintained by various scripts. In this case, this repo has two grammar files and the `add-grammar` script will have added two lines to this file:

```suggestion
- source.vba
- source.wwb
```

---

Thanks! Weirdly, I did run the `add-grammar` script on a separate branch for comparison before submitting the PR for review, but it didn't add the lines to `grammar.yml`. However, I cleaned the repo this morning, tried it again and the lines were added this time 🤷‍♂️.